website/docs/r/host_virtual_switch.html.markdown

```diff
@@ -174,3 +174,16 @@ the resource. This is set to an ID value unique to Terraform - the convention
 is a prefix, the host system ID, and the virtual switch name. An example would
 be `tf-HostVirtualSwitch:host-10:vSwitchTerraformTest`.
 
+## Importing
+
+An existing vSwitch can be [imported][docs-import] into this resource via the id
```

Comment:
The `id` for vswitches isn't very intuitive. I think having an explaination that the ID is the <prefix (tf-HostVirtualSwitch)>:<host MOID>:<switch name> would help make it more clear. Sorry the `id` isn't simpler in this case!